suites/nautilus/ansible/sanity_ceph_ansible_scale.yaml

```diff
@@ -0,0 +1,200 @@
+              - name: "cephfs_data"
+                pgs: "8"
+              - name: "cephfs_metadata"
+                pgs: "8"
+      desc: osd with 6 osd scenarios with lvm
+      destroy-cluster: False
+      abort-on-fail: true
+
+   - test:
+      name: config roll over rgw daemon
+      polarion-id: CEPH-9581
+      module: test_ansible_roll_over.py
+      config:
+          is_mixed_lvm_configs: True
+          ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_stable_release: nautilus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              journal_size: 1024
+              ceph_stable: True
+              ceph_stable_rh_storage: True
+              fetch_directory: ~/fetch
+              copy_admin_key: true
+              dashboard_enabled: False
+              ceph_conf_overrides:
+                  global:
+                    osd_pool_default_pg_num: 64
+                    osd_default_pool_size: 2
+                    osd_pool_default_pgp_num: 64
+                    mon_max_pg_per_osd: 1024
+                  mon:
+                    mon_allow_pool_delete: true
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "8"
+                - name: "cephfs_metadata"
+                  pgs: "8"
+          add:
+              - node:
+                  node-name: .*node4.*
+                  demon:
+                      - rgw
+      desc: add rgw daemon
+
+   - test:
+      name: config roll over mds daemon
+      polarion-id: CEPH-9581
+      module: test_ansible_roll_over.py
+      config:
+          is_mixed_lvm_configs: True
+          ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_stable_release: nautilus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              journal_size: 1024
+              ceph_stable: True
+              ceph_stable_rh_storage: True
+              fetch_directory: ~/fetch
+              copy_admin_key: true
+              dashboard_enabled: False
+              ceph_conf_overrides:
+                  global:
+                    osd_pool_default_pg_num: 64
+                    osd_default_pool_size: 2
+                    osd_pool_default_pgp_num: 64
+                    mon_max_pg_per_osd: 1024
+                  mon:
+                    mon_allow_pool_delete: true
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "8"
+                - name: "cephfs_metadata"
+                  pgs: "8"
+          add:
+              - node:
+                  node-name: .*node5.*
+                  demon:
+                      - mds
+      desc: add mds daemon
+
+   - test:
+      name: config roll over client daemon
+      polarion-id: CEPH-9581
+      module: test_ansible_roll_over.py
+      config:
```

Comment:
Is ansible config(ansi_config) parameters are common across all playbook and considered at every execution?. if not used, can we eliminate it?

Reply:
https://github.com/red-hat-storage/cephci/blob/master/tests/ceph_ansible/test_ansible_roll_over.py#L66 yes they are common and considered for every execution

Reply:
Do we need ansible_config even after deployment? other than only few parameters which is required for upgrade or to run specific playbook.